lib/wallaby/phantom.ex

```diff
@@ -50,6 +52,36 @@ defmodule Wallaby.Phantom do
     :poolboy.checkin(Wallaby.ServerPool, server)
   end
 
+  defdelegate accept_dialogs(session),                            to: Driver
+  defdelegate accept_alert(session, open_dialog_fn),              to: Driver
+  defdelegate accept_confirm(session, open_dialog_fn),            to: Driver
+  defdelegate accept_prompt(session, input_va, open_dialog_fn),   to: Driver
+  defdelegate cookies(session),                                   to: Driver
+  defdelegate current_path!(session),                             to: Driver
+  defdelegate current_url!(session),                              to: Driver
+  defdelegate dismiss_dialogs(session),                           to: Driver
+  defdelegate dismiss_confirm(session, open_dialog_fn),           to: Driver
+  defdelegate dismiss_prompt(session, open_dialog_fn),            to: Driver
+  defdelegate get_window_size(session),                           to: Driver
+  defdelegate page_title(session),                                to: Driver
+  defdelegate page_source(session),                               to: Driver
+  defdelegate set_cookies(session, key, value),                   to: Driver
+  defdelegate set_window_size(session, width, height),            to: Driver
+  defdelegate visit(session, url),                                to: Driver
+
+  defdelegate attribute(element, name),                           to: Driver
+  defdelegate click(element),                                     to: Driver
+  defdelegate clear(element),                                     to: Driver
+  defdelegate displayed(element),                                 to: Driver
+  defdelegate selected(element),                                  to: Driver
+  defdelegate set_value(element, value),                          to: Driver
+  defdelegate text(element),                                      to: Driver
+
+  defdelegate execute_script(session_or_element, script, args),   to: Driver
+  defdelegate find_elements(session_or_element, compiled_query),  to: Driver
+  defdelegate send_keys(session_or_element, keys),                to: Driver
+  defdelegate take_screenshot(session_or_element),                to: Driver
```

Comment:
Check out the last commit on [this branch](https://github.com/aaronrenner/wallaby/commits/ar-selenium-for-real) to see the what the selenium driver looks like. In that case, I like all of the functions necessary to build a driver being on a single, top-level module. The reason I used delegates was to make `Wallaby.Phantom` conform to the same interface without having to move a ton of code at the moment.

I do realize that I have a lot of functions in my test Selenium driver that could be converted to delegates at the moment
```elixir
def text(%Element{} = element) do
  WebdriverClient.text(element)
end
```
but that's only because I've extracted all of the http calls into a Selenium specific webdriver-client. The interface between `Wallaby.Experimental.Selenium` and `Wallaby.Experimental.Selenium.WebdriverClient` is still fluid and eventually won't match up exactly with what's expected by the `Wallaby.Driver` behaviour. My goal is for `Wallaby.Phantom.Driver` to be replaced by extracting a phantom specific api client, plus moving additional logic like checking logs into `Wallaby.Phantom`.